tasks/managed/add-fbc-contribution/add-fbc-contribution.yaml

```diff
@@ -91,6 +91,7 @@ spec:
         else
           iib_service_account_secret="iib-service-account-prod"
         fi
+        publishing_credentials=$(jq -r '.fbc.publishingCredentials // "catalog-publishing-secret"' "$DATA_FILE")
```

Comment:
This is fine, but I think it is kind of weird having the default in both here and the internal pipeline parameter default. The pipeline default won't matter, only this one will, but still a little confusing if they get out of sync. Maybe remove the default in one of the two spots?

Reply:
The thing is that the default in the other place won't really work without extra work here (checking for null or empty) - if you just remove this default here (which I suggested adding), then you would default to "null". So yeah, I'd just remove the default in the pipeline.

Reply:
ok. I removed the default value from the pipeline.